### ISSUE TITLE
Disable screen reader detection?

### ISSUE THREAD

**Issue by crlf0710** [Microsoft/vscode#29087]
- VSCode Version: Code 1.13.1 (379d2efb5539b09112c793d3d9a413017d736f89, 2017-06-14T18:21:47.485Z)
- OS Version: Windows_NT ia32 10.0.14393
- Extensions: 

|Extension|Author|Version|
|---|---|---|
|javadebugger|donjayamanne|0.1.2|
|python|donjayamanne|0.6.5|
|mediawiki|jakeboone02|2.1.0|
|vscode-rust|kalitaalexey|0.3.14|
|cpptools|ms-vscode|0.11.4|
|prolog|rebornix|0.0.4|
|java|redhat|0.6.0|
|vscoq|siegebell|0.2.7|
|vscode-react-native|vsmobile|0.3.2|
|debug|webfreak|0.20.0|
|Idris|zjhmale|0.9.8|;

---
Hi, i've got screen reader software installed on my system. They're needed for work. But usually i'm not using them. (I've disabled it from the control panel).  However, i found that VS Code always say "Screen reader detected" and word wrapping is always disabled, which is a little annoying.

Anything i can do to disable the vs code accessibility support? Thanks a lot!

**Comment by alexandrudima** [comment]
@crlf0710 If you hover for 3s over the "Screen reader detected", there should be a pop up guiding you towards disabling the feature. Do you have any idea how to make the hover more discoverable? TL;DR There is a setting `editor.accessibilitySupport` that can be set to `off`.

<img width="476" alt="screen shot 2017-06-21 at 09 36 16" src="https://user-images.githubusercontent.com/5047891/27372474-1c9d999e-5665-11e7-93d5-c6296825ca1d.png">
